src/components/admin/form-components/validated-fields.jsx

```diff
@@ -0,0 +1,15 @@
+import TextField from 'material-ui/TextField';
+
+// Higher Order Components
+import withValidate from 'components/admin/form-components/withValidate';
+
+export const hasHttps = (value) => {
+  if (value && value.length > 4 && value.substr(0, 5) !== 'https') {
+    return (
+      'You are saving an image without using https.'
+    );
+  }
+  return null;
```

Comment:
What's the reason for using null here and not false?

Reply:
I guess it makes more semantic sense in either it's a string or null, it kinda keeps it within a single type as null can be thought of as any type

Reply:
Since Javascript is good at truthiness, `null` and false are effectively the same for the current use case and this preserves the fact that `validator(value)` is always a string. If I made it false, if I did something like `<h1>{validator(value)}</h1>` I would sometimes end up with `<h1>false</h1>` which is confusing compared to just getting `<h1></h1>`